atef/report.py

```diff
@@ -177,7 +177,7 @@ def build_passive_summary_table(story: List[Flowable], prep_file: PreparedFile):
         if isinstance(item, PreparedConfiguration):
             name = item.config.name
         elif isinstance(item, PreparedComparison):
-            name = item.comparison.name + ' - ' + item.identifier
+            name = str(item.comparison.name) + ' - ' + str(item.identifier)
```

Comment:
Nitpick, but it's just so much easier to read:
```suggestion
            name = f"{item.comparison.name} - {item.identifier}"
```